src/awkward/behaviors/mixins.py

```diff
@@ -63,7 +63,7 @@ def register(cls):
     return register
 
 
-def mixin_class_method(ufunc, rhs=None, transpose=True):
+def mixin_class_method(ufunc, rhs=None, *, transpose=True):
```

Comment:
This is one that I should ask @nsmith- about: is it the case that `rhs` is likely to be positional but `transpose` should always be written out as keyword-only?

Reply:
Okay, good to know about `rhs`.

About `transpose`: changing the default from `True` to `False` would be dangerous, even at the 2.0.0 boundary, unless we also change the name. As a keyword-only argument, the written-out old name would be different from the written-out new name, so old code can't pass through accidentally. After the 2.0.0 boundary, we'd have to have a period of time in which both names exist, with a warning on the old name, but we could make a clean switch to a new name if we do it before the 2.0.0 boundary.

What do you think? Is there a better word than `transpose` that would justify the change in default?

Reply:
I think this will just have to stay. It is not a well-trodden path anyway.